pkg/tenant/members_test.go

```diff
@@ -12,6 +12,35 @@ import (
 	"github.com/trisacrypto/directory/pkg/trtl/pb/v1"
 )
 
+func (suite *tenantTestSuite) TestTenantMemberList() {
+	require := suite.Require()
+	ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
+
+	defer cancel()
+
+	// Connect to mock trtl database.
+	trtl := db.GetMock()
+	defer trtl.Reset()
+
+	// Call the OnCursor method.
+	trtl.OnCursor = func(cr *pb.CursorRequest, t pb.Trtl_CursorServer) error {
+		return nil
+	}
+
+	req := &api.PageQuery{
+		PageSize:      2,
+		NextPageToken: "12",
+	}
+
+	// Should return an error if the tenant does not exist.
+	_, err := suite.client.TenantMemberList(ctx, "invalid", req)
+	suite.requireError(err, http.StatusBadRequest, "could not parse tenant ulid", "expected error when tenant does not exist")
+
+	members, err := suite.client.TenantMemberList(ctx, "01ARZ3NDEKTSV4RRFFQ69G5FAV", req)
+	require.NoError(err, "could not list tenant members")
+	require.Len(members.TenantMembers, 1, "expected one member in the database")
```

Comment:
Do we also need to test that we get back the specific member that we're expecting (in addition to making sure we get the right # of members back)?

Reply:
I'll look into this more on how to add that test.